plugins/by-name/schemastore/default.nix

```
@@ -128,15 +128,24 @@ lib.nixvim.plugins.mkVimPlugin {
         message = "You have enabled the plugin, but neither `json` or `yaml` schemas are enabled.";
       }
       {
-        when = !(cfg.json.enable -> config.plugins.lsp.servers.jsonls.enable);
-        message = "You have enabled `json` schemas, but `plugins.lsp.servers.jsonls` is not enabled.";
+        when =
+          cfg.json.enable && !(config.plugins.lsp.servers.jsonls.enable || config.lsp.servers.jsonls.enable);
+        message = ''
+          You have enabled `json` schemas, but neither
+          `plugins.lsp.servers.jsonls` nor `lsp.servers.jsonls` is enabled.
```

Nit: as-per https://github.com/nix-community/nixvim/pull/3262#discussion_r2072214733, we should wrap lines as we want them to appear in stderr output, not as we want them to appear in code.

```suggestion
          You have enabled `json` schemas, but neither `plugins.lsp.servers.jsonls` nor `lsp.servers.jsonls` is enabled.
```